test/async-hooks/test-zctx.js

```diff
@@ -0,0 +1,31 @@
+const common = require('../common')
+const assert = require('assert')
+const initHooks = require('./init-hooks')
+const { checkInvocations } = require('./hook-checks')
+const zlib = require('zlib')
+
+const hooks = initHooks()
+
+hooks.enable()
+zlib.deflate('..........', common.mustCall(ondeflate));
+
+function ondeflate() {
+  const as = hooks.activities()
```

Comment:
Please use `activitiesOfTypes` here like I do in [statwatcher](https://github.com/thlorenz/node/blob/trevnorris-async-wrap-eps-impl%2Btests/test/async-hooks/test-statwatcher.js#L16) for instance.
This makes the tests less brittle .. I yet have to update some tests that I wrote earlier to use that method as well.